sources/System/IntroAnimation/IntroAnimationSystem.cpp

```diff
@@ -25,25 +26,47 @@ namespace BBM
         auto scene = wal.getScene();
         RAY2D::Rectangle *rectangle = nullptr;
         RAY2D::Text *text = nullptr;
+        static auto &powered_text = scene->addEntity("powered by text")
+			.addComponent<PositionComponent>(1920 / 2 - 200, 1080 / 2 - 180, 0)
+			.addComponent<Drawable2DComponent, RAY2D::Text>("powered by", 30, RAY::Vector2(), WHITE);
         static auto &rayText = scene->addEntity("raylibtext Rectangle")
             .addComponent<PositionComponent>(1920 / 2 - 44, 1080 / 2 + 48, 0)
-            .addComponent<Drawable2DComponent, RAY2D::Text>("", 50, RAY::Vector2(), BLACK);
+            .addComponent<Drawable2DComponent, RAY2D::Text>("", 50, RAY::Vector2(), WHITE);
         static auto &bottomRectangle = scene->addEntity("bottom Rectangle")
             .addComponent<PositionComponent>(1920 / 2 - 128, 1080 / 2 - 128, 0)
-            .addComponent<Drawable2DComponent, RAY2D::Rectangle>(logoPos, RAY::Vector2(16, 16), BLACK);
+            .addComponent<Drawable2DComponent, RAY2D::Rectangle>(logoPos, RAY::Vector2(16, 16), WHITE);
         static auto &leftRectangle = scene->addEntity("left Rectangle")
             .addComponent<PositionComponent>(1920 / 2 - 128, 1080 / 2 - 128, 0)
-            .addComponent<Drawable2DComponent, RAY2D::Rectangle>(logoPos, RAY::Vector2(16, 16), BLACK);
+            .addComponent<Drawable2DComponent, RAY2D::Rectangle>(logoPos, RAY::Vector2(16, 16), WHITE);
         static auto &rightRectangle = scene->addEntity("right Rectangle")
             .addComponent<PositionComponent>(1920 / 2 - 128, 1080 / 2 - 128, 0)
-            .addComponent<Drawable2DComponent, RAY2D::Rectangle>(logoPos, RAY::Vector2(16, 16), BLACK);
+            .addComponent<Drawable2DComponent, RAY2D::Rectangle>(logoPos, RAY::Vector2(16, 16), WHITE);
         static auto &topRectangle = scene->addEntity("top Rectangle")
             .addComponent<PositionComponent>(1920 / 2 - 128, 1080 / 2 - 128, 0)
-            .addComponent<Drawable2DComponent, RAY2D::Rectangle>(logoPos, RAY::Vector2(16, 16), BLACK);
+            .addComponent<Drawable2DComponent, RAY2D::Rectangle>(logoPos, RAY::Vector2(16, 16), WHITE);
+        static auto &epitechLogo = scene->addEntity("epitech logo")
+			.addComponent<PositionComponent>(1920 / 2 - 150, 1080 / 2 - 61, 0)
+			.addComponent<Drawable2DComponent, RAY::Texture>("assets/epitech.png");
         static short letterCounter = 0;
 
         switch (component.currentStep)
         {
+        case IntroAnimationComponent::animationSteps::epitechLogo:
```

Comment:
Is it normal that we don't see an epitech logo while launching the game?

Reply:
I see it

Reply:
I didn't pull. Oups